test/test_get_soup.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import mock
+import pytest
+import sys
+
+sys.modules["config"] = mock.MagicMock()
+from amz_review_scraper.get_soup import boil_soup
+
+
+def mocked_requests_get(*args, **kwargs):
```

Comment:
I would move this to a separate file, and import it. With tests they end up being like documentation, and you want the test files to tell as clear a story as possible.

`test/support/mock_request.py` is a potential path.